src/popup/pages/Send/index.js

```diff
@@ -869,6 +869,7 @@ class SendPage extends React.Component {
           />
           <Button
             content={getLanguage('confirm')}
+            disabled
```

Comment:
hmm, removing this in devtools doesn't fix click

Reply:
how does `disabled` work in this component?

Reply:
oh it goes into `<button disabled={disabled}>`

Reply:
`Button` just forwards it to `button`, then react does a lot of magic that I don't know.
Either react doesn't attach the listener when it creates a disabled button, or it ignores events based on internal props without looking at changed html.

But this is probably involved https://github.com/facebook/react/blob/9abe745/packages/react-dom/src/events/getListener.js#L40